docs/operations/python.md

```diff
@@ -0,0 +1,46 @@
+---
+id: python
+title: "Python Installation"
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+Apache Druid startup script requires a python interpreter.
+
+## Python interpreter installation instructions
+
+### MacOS
+MacOS comes with a version of Python2, which has been deprecated. Python2 and Python3 can coexist on the same machine without problems.
```

Comment:
I don't think this is accurate anymore; see https://www.jamf.com/blog/python2-is-gone-apple-macos/. The python situation on macOS is evolving. We should do our best to describe how to get a working interpreter on the 3–4 most recent major macOS releases, which will involve different instructions for different macOS releases.